debian/kano-desktop.install

```diff
@@ -1,11 +1,9 @@
-#!/usr/bin/dh-exec
```

Comment:
Probably also remove the execution bit from the file if not already, but no big deal

Reply:
Good point! I completely forgot about that part 👍